### PR TITLE
DOC: updated .combine_first doc strings

### PR DESCRIPTION
**Pull request by topper-123**
This PR updates ``Series.combine_first``/``DataFrame.combine_first`` doc strings. In Particular, it add some short examples to the doc strings.

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -4125,16 +4125,23 @@ def combine_first(self, other):
...
+        >>> df1.combine_first(df2)
+           0    1
+        0  1  4.0
 
-        Returns
-        -------
-        combined : DataFrame
+        See Also
+        --------
+        DataFrame.combine
```
Comment (jorisvandenbossche) [inline pandas/core/frame.py]: Can you put a short sentence here highlighting the difference?

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks!

Added a small comment

File: pandas/core/frame.py (hunk 1 above)
Comment (topper-123) [reply pandas/core/frame.py]: Added

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**